src/veh_type.cpp

```diff
@@ -280,6 +280,8 @@ void vpart_info::finalize()
     }
 
     for( auto& e : vehicle_part_types ) {
+        e.second.name_ = _( e.second.name_.c_str() );
```

Comment:
`vpart_info::name()` indicates that it is possible to have an empty `name_` (the name in JSON seems to be optional). This would mean `e.second.name_` can be empty, translating it will result in the translation project info string, not an empty string again. Or can `e.second.name_` not actually be empty?

Btw. why not move the logic from `name()` here? The item data should already be available here. And `name_` can be made non-mutable.